packages/p2p/src/components/app.scss

```
@@ -56,6 +56,26 @@
             padding: 3.5rem 2.5rem 0;
         }
     }
+    &__advertiser-tab {
+        @include mobile {
+            .dc-tabs {
+                &__item {
+                    padding: 0px 1.6rem;
```

You don't need `0px` here.

```suggestion
                    padding: 0 1.6rem;
```